src/plugins/intel_cpu/tests/unit/gemm_api_test.cpp

```diff
@@ -0,0 +1,14 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <utility>
+#include <gtest/gtest.h>
+#include "gemm/ov_cpu_gemm.h"
+
+
+TEST(GemmTests, getPackedSize) {
```

Comment:
Could you please name this test (and maybe add some comment)
To be honest it is hard to understand why we need this test at all

Reply:
Hi, this test is used to check whether the mlas is successfully compiled and integrated into OV